src/routes/Settings.tsx

```diff
@@ -100,13 +118,119 @@ const Settings: React.FC = () => {
+              <Tag key={subject}>{subject.toUpperCase()}</Tag>
+            ))}
+          </>
+        ),
+      },
+      {
+        title: 'Status',
+        key: 'status',
+        filters: ['manual', 'awaiting-approval', 'approved'].map((s) => ({
+          value: s,
+          text: stateTranslation[s],
+        })),
+        filterMultiple: false,
+        onFilter: (value: string, certificate: IExposedCertificate) =>
+          certificate.state === value,
+        // defaultSortOrder: 'descend',
+        sorter: (a: IExposedCertificate, b: IExposedCertificate) =>
+          a.state.length - b.state.length,
+        // sortDirections: ['descend', 'ascend'],
+        render: (certificate: IExposedCertificate) => (
+          <>
+            <Tag
+              color={
+                (certificate.state === 'manual' && 'green') ||
+                (certificate.state === 'awaiting-approval' && 'yellow') ||
+                (certificate.state === 'approved' && 'green')
+              }
+              key={certificate.state}
+            >
+              {stateTranslation[certificate.state]}
+            </Tag>
+          </>
+        ),
+      },
+      {
+        title: 'Aktion',
+        key: 'action',
+        render: (certificate: IExposedCertificate) => (
+          <Space size="middle">
+            {userContext.user.isTutor &&
+              certificate.state !== 'awaiting-approval' && (
+                <>
+                  <Select
+                    defaultValue={defaultLanguage}
+                    onChange={(event) => setLanguage(event)}
+                    style={{ width: 120 }}
+                  >
+                    {Object.entries(supportedLanguages).map(([code, value]) => (
+                      <Select.Option value={code}>{value}</Select.Option>
+                    ))}
+                  </Select>
+                  <Button
+                    type="primary"
+                    onClick={() => showCertificate(certificate.uuid)}
+                  >
+                    Ansehen
+                  </Button>
+                </>
+              )}
+
+            {/* <Button danger>Löschen</Button> */}
+          </Space>
+        ),
+      },
+    ];
+
+    return (
+      <>
+        <Title size="h3" className={classes.subjectTitle}>
+          Zertifikate
+        </Title>
+        <Table dataSource={certificates.value ?? []} columns={columns} />
+      </>
+    );
+  };
+
   return (
     <div className={classes.container}>
       <Title>Deine Informationen</Title>
-      <SettingsCard user={userContext.user} />
+      <SettingsCard
+        user={userContext.user}
+        reloadCertificates={reloadCertificates}
+      />
       {!isProjectCoachButNotTutor(userContext.user) && renderSubjects()}
       {(userContext.user.isProjectCoach || userContext.user.isProjectCoachee) &&
         renderProjectFields()}
+      {renderCertificatesTable()}
       <AccountNotScreenedModal />
     </div>
```

Comment:
![image](https://user-images.githubusercontent.com/33378936/102548862-89ce6900-40bb-11eb-8b35-f782a4c54aab.png)
I think it would be nicer if we had a margin on the right of the table ;-)